en/mkdocs.yml

```diff
@@ -361,6 +361,20 @@ nav:
     - 'Performance':
       - 'Performance Tuning Recommendations': deploy/performance/performance-tuning-recommendations.md
       - 'Configure Cache Layers': deploy/performance/configuring-cache-layers.md
+    - 'Migrate':
+      - 'Migration Process': deploy/migrate/migration-process.md
+      - 'What has Changed': deploy/migrate/migrating-what-has-changed.md
+      - 'Migration Guide':
+        - 'Before you Begin': deploy/migrate/migration-guide.md
+        - 'Prepare for Migration': deploy/migrate/migrating-preparing-for-migration.md
+        - 'Migrate to 5.11.0': deploy/migrate/migrating-to-5110.md
+        - 'Migrate Userstore Managers': deploy/migrate/migrating-userstore-managers.md
+      - 'Feature Deprecation': deploy/migrate/wso2-identity-server-feature-deprecation.md
+    - 'Deploy WSO2 Using':
```

Comment:
Shall we change this topic to something like "Deployment Options" or a more suitable topic